### DISCUSSION TITLE
误删除配置怎么才能恢复？

### DISCUSSION THREAD

**Discussion by xiaoyuit** [AlistGo/alist#2561] [Q&A] [answered]
请问不小心删除了【设置】->【站点】下面可删除的配置，怎么才能恢复？
![image](https://user-images.githubusercontent.com/5516423/205367425-7f36d31f-c2cb-473b-b602-6f555b41b6f6.png)

**Comment by xhofe** [comment, marked as answer]
带删除按钮的表示被弃用的设置 不用管

**Reply by xiaoyuit** [reply]
好的，谢谢了
